### PR TITLE
Database Wireframe Update

### PR DESCRIPTION
**Pull request by anderscc**

### CODE REVIEW

File: server/scheduler/models.py [hunk 1]
```diff
@@ -39,7 +39,13 @@ class Labs(models.Model):
 class Assignment(models.Model):
     scheduleNum = models.AutoField(primary_key = True, editable = False, unique = True)
     semYr = models.ForeignKey('SemesterYear', on_delete=models.CASCADE)
-    id = models.ForeignKey('GATA', on_delete=models.CASCADE)
+    id = models.ForeignKey('GATA', on_delete=models.CASCADE) #GA id which is refernced from GATA
```
Comment (Caleb7621-java) [inline server/scheduler/models.py:42]: These id variables need to be distinguished. You are currently overwriting the first instance of "id". These should be named GAid and TAid.

Comment (Caleb7621-java) [inline server/scheduler/models.py:42]: These id variables need to be distinguished. You are currently overwriting the first instance of "id". These should be named GAid and TAid.

Comment (Caleb7621-java) [inline server/scheduler/models.py]: These id variables need to be distinguished. You are currently overwriting the first instance of "id". These should be named GAid and TAid.

**Review by Caleb7621-java — CHANGES_REQUESTED** [review]
These id variables need to be distinguished. You are currently overwriting the first instance of "id". These should be named GAid and TAid.

**Review by Caleb7621-java — APPROVED** [review]
Changes have been made.

**Review by zhao926 — APPROVED** [review]

**Pull request merged**